src/system.c

```diff
@@ -1519,8 +1512,51 @@ static void SySetGapRootPath( const Char * string )
         n++;
 #endif
     }
+    return;
 }
 
+/****************************************************************************
+**
+*F  SySetInitialGapRootPaths( <string> )  . . . . .  set the root directories
+**
+**  Setup GAP's initial root paths, based on two sources:
+**  1) If the environment variable GAPROOT is undefined, the
+**     C macro GAPROOT_LOCATION (defined when GAP was compiled).
+**  2) Alternatively, try the location of GAP's executable.
+*/
+static void SySetInitialGapRootPaths(void)
+{
+
+#ifdef GAPROOT_LOCATION
+    SySetGapRootPath(GAPROOT_LOCATION);
```

Comment:
I may have asked this before but couldn't find the answer: Why add `GAPROOT_LOCATION`? Doesn't it do exactly the same thing as the existing `SYS_DEFAULT_PATHS`?